pep_sphinx_extensions/pep_zero_generator/writer.py

```diff
@@ -169,6 +169,13 @@ def write_pep0(
                 self.emit_newline()
             self.emit_newline()
 
+            self.emit_title("API")
```

Comment:
Maybe we don't want this section near the top of the PEP 0 page?

Reply:
I think it's fine here as it's very short.